examples/fishjam-chat/app.json

```diff
@@ -13,7 +13,12 @@
     },
     "ios": {
       "supportsTablet": true,
-      "bundleIdentifier": "io.fishjam.example.fishjamchat"
+      "bundleIdentifier": "io.fishjam.example.fishjamchat",
+      "infoPlist": {
+        "NSAppTransportSecurity": {
+          "NSAllowsArbitraryLoads": true
```

Comment:
Can we add some comment that is required for testing? I'm afraid that some devs might think it is required for multimedia streaming.
(or maybe - can we move it to our expo dev plugin?)

Reply:
Not sure it is our business. The reason why i added this is because in the same network i may have not secure server. In general that setting is very dangerous and should be ommited in release apps by devs. Here is more resources: https://developer.apple.com/documentation/bundleresources/information_property_list/nsapptransportsecurity/nsallowsarbitraryloads

Reply:
I will add comment about it.